src/App.js

```diff
@@ -10,7 +10,7 @@ const App = () => (
       <Header>
         <Routes>
           <Route path="/" element={<Home />} />
-          <Route path="/details" element={<Details />} />
+          <Route path="/:id" element={<Details />} />
```

Comment:
- [ ] Great job. Please kindly follow the Behance grid design. Your version has gaps between cards, and that is not the case in the Behance design.
- [ ] also, some of the text in your cards are overlapping. Please fix that 
yours
![Screenshot (531)_LI](https://user-images.githubusercontent.com/65298048/195947351-7fb5331f-e1ab-4347-a487-8b693d017550.jpg)
Behance
![Screenshot (532)](https://user-images.githubusercontent.com/65298048/195947423-183e6dcb-6c11-468a-a3cc-d871363788d9.png)